.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
-node_modules
\ No newline at end of file
+build
```

Comment:
The `tsconfig.json` file specifies `build` for the `outDir` option. The only thing in there is `tsconfig.tsbuildinfo` but just to be safe I've added it to the list of ignored files.